src/config.ts

```diff
@@ -16,8 +17,10 @@ export const bumpConfigDefaults: VersionBumpOptions = {
 
 export async function loadBumpConfig(overrides?: Partial<VersionBumpOptions>,
   cwd = process.cwd()) {
+  const childrenNames = readdirSync(cwd)
+  const name = childrenNames.some(name => name.includes('bumpp')) ? 'bumpp' : 'bump'
   const { config } = await loadConfig<VersionBumpOptions>({
-    name: 'bump',
+    name,
```

Comment:
I don't think this is a good idea to rely on readdirSync, as the config can come from different sources. Instead we could run c12 twice and merge the config.

Reply:
Yes, I missed that point.